Asterix/Main_EFC_THD.py

```diff
@@ -524,92 +413,45 @@ def correctionLoop(parameter_file,
     thd2.entrancepupil = pup_round
     thd2.DM1 = DM1
     thd2.DM3 = DM3
-    thd2.corono = corono
     # In practice this is done inside the Testbed initialization already !
     # and these lines are useless and I only put them to calm
     # To be removed when the correction and estimation class are done
     # in which case these class will be able when these things exists or not
 
-    PSF = thd2.todetector_Intensity(center_on_pixel=True, noFPM=True)
-    thd2.maxPSF = np.max(PSF)
-    thd2.sumPSF = np.sum(PSF)
-
-    #image size after binning
-    dim_sampl = int(DH_sampling / thd2.science_sampling * thd2.dim_im / 2) * 2
-
-    if onbench == True:
-        Labview_dir = os.path.join(Data_dir, "Labview") + os.path.sep
-        if not os.path.exists(Labview_dir):
-            print("Creating directory " + Labview_dir + " ...")
-            os.makedirs(Labview_dir)
-
-    result_dir = Labview_dir = os.path.join(Data_dir, "Results",
-                                            Name_Experiment) + os.path.sep
+    result_dir = os.path.join(Data_dir, "Results",
+                              Name_Experiment) + os.path.sep
     if not os.path.exists(result_dir):
         print("Creating directory " + result_dir + " ...")
         os.makedirs(result_dir)
 
     ## Pair-wise probing directory
     if DM3_otherbasis == True:
         basistr = "fourier"
-        DM3_basis = fits.getdata(Labview_dir + "Map_modes_DM3_foc.fits")
-        thd2.DM3.WhichInPupil = np.arange(thd2.DM3.DM_pushact.shape[0])
+        DM3_basis = fits.getdata(result_dir + "Map_modes_DM3_foc.fits")
```

Comment:
Are you sure about the directory path? Was there a reason why it was Labview_dir here?

Reply:
Not sure, it was like that but Map_modes_DM3_foc.fits is not in the git hub right now, so this is not functionnal.

"other basis" system need to be carefully redone, this is currently a mess.